src/doc/en/developer/packaging.rst

```diff
@@ -1079,7 +1079,7 @@ For Python packages available from PyPI, there is another shortcut::
     [...............................................................]
 
 The ``upstream_url`` information serves yet another purpose.
-Developers who wish to test a package update from a Trac branch before
+Developers who wish to test a package update from a Issue branch before
 the archive is available on a Sage mirror can do so by configuring
```

Comment:
Change "Issue branch" to "PR"

Reply:
I'm leaving this untouched for now (we need to update documentation in a separate PR).